RoaringBitmap/src/main/java/org/roaringbitmap/longlong/Roaring64Bitmap.java

```diff
@@ -334,6 +334,40 @@ public void or(final Roaring64Bitmap x2) {
     }
   }
 
+  /**
+   * Bitwise OR (union) operation. The provided bitmaps are *not* modified. This operation is
+   * thread-safe as long as the provided bitmaps remain unchanged.
+   *
+   * @param x1 first bitmap
+   * @param x2 other bitmap
+   * @return result of the operation
+   */
+  public static Roaring64Bitmap or(final Roaring64Bitmap x1, final Roaring64Bitmap x2) {
```

Comment:
Should the implementation be subsentially more similar to `https://github.com/RoaringBitmap/RoaringBitmap/blob/master/RoaringBitmap/src/main/java/org/roaringbitmap/RoaringBitmap.java#L2471` ?

Reply:
Yes, I agree. It's been modified.